src/components/Alert/Alerts.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import {Alert} from 'react-bootstrap';
+import styles from './Alert.module.scss';
+
+interface Props {
+  type: string;
+}
```

Comment:
I recommend adding type information to the type prop.

```suggestion
type AlertNotificationTypes = 'info' | 'warning' | 'notice';

interface Props {
  type: AlertNotificationTypes;
}
```